client/components/flag/index.jsx

```diff
@@ -15,11 +15,16 @@ const Flag = React.createClass( {
 	render() {
 		const props = omit( this.props, [ 'icon', 'type', 'className' ] );
 
+		var noticon;
```

Comment:
Should be using `let` here. More importantly, though, we should drop reliance on noticons in favor of gridicons.